src/policy/concrete.rs

```diff
@@ -264,6 +349,30 @@ impl<Pk: MiniscriptKey> Policy<Pk> {
         }
     }
 
+    /// Translate `Semantic::Key(key)` to `Semantic::Unsatisfiable` when extracting TapKey
+    pub fn translate_unsatisfiable_pk(&self, key: &Pk) -> Policy<Pk> {
```

Comment:
This is can consume Self.

Reply:
Done in [e50766](https://github.com/rust-bitcoin/rust-miniscript/pull/291/commits/e507066a5288c09c098d5eac9980066555e287ce).